OSSM-8770 Update feature support tables (#88965)
openshift/openshift-docs: modules/ossm-release-notes-kubernetes-gateway-api.adoc

////
Module included in the following assemblies:
* service-mesh-docs-main/ossm-release-notes-support-tables-assembly.adoc
////

//Post GA: might be worth considering putting tables into their own modules. Might easier to maintain. Warrants further discussion post GA.

:_mod-docs-content-type: REFERENCE
[id="kubernetes-gateway-api_{context}"]
= Kubernetes Gateway API

[cols="1,1"]
|===
| Feature | Status

| Kubernetes Gateway APIs for ingress (Gateway parentRef)
| GA

| Kubernetes Gateway APIs for mesh (Service parentRef)
| GA

> | Kubernetes Gateway API Custom Resource Definitions (CRDs)

Reviewer: ```suggestion
| Kubernetes Gateway API custom resource definitions (CRDs)
```

| DP ^[1]^

> | Kubernetes Gateway API Manual Deployment

Reviewer: Same, can this be lower? And to check any that I didn't call out

```suggestion
| Kubernetes Gateway API manual deployment
```

| NA

| Gateway Network Topology configuration
| DP
|===

> . The use of Kubernetes Gateway API requires custom resource definitions (CRDs) that are not installed with Red Hat OpenShift 4.18 and earlier releases.

Reviewer: Just checking if it has to be "Red Hat OpenShift", or if it should be {ocp-product-title}

```suggestion
. The use of Kubernetes Gateway API requires custom resource definitions (CRDs) that are not installed with {ocp-product-title} 4.18 and earlier releases.
```
